containers/runtime/ruby/Dockerfile

```
@@ -0,0 +1,24 @@
+# Copyright 2020 gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM ruby:2.5
+
+RUN mkdir -p /src/workspace
+WORKDIR /src/workspace
+
+RUN apt-get update && apt-get install -y \
+  git\
+  time && \
+  apt-get clean
+
+CMD ["bash"]
```

GitHub is complaining b/c this file does not end in 1 blank new line. I think that's a linux convention. I'd add it just to shut it up. Otherwise, LGTM.

```suggestion
CMD ["bash"]

```